doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,895 @@
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.4.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.4.0.
+
+:ref:`Toolchain Manager <gs_app_tcm>`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.4.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.4.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_240_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE and tool support
+====================
+
+|no_changes_yet_note|
+
+MCUboot
+=======
+
+|no_changes_yet_note|
+
+Application development
+=======================
+
+|no_changes_yet_note|
+
+RF Front-End Modules
+--------------------
+
+|no_changes_yet_note|
+
+Build system
+------------
+
+* When using :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` with relative paths, the relative path is now points to the application configuration directory instead of the application source directory (these are the same if the application configuration directory is not set).
+
+Working with nRF52 Series
+=========================
+
+|no_changes_yet_note|
+
+Working with nRF53 Series
+=========================
+
+|no_changes_yet_note|
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth® LE
+-------------
+
+|no_changes_yet_note|
+
+Bluetooth mesh
+--------------
+
+* Updated:
+
+  * The protocol user guide with the information about :ref:`ble_mesh_dfu_samples`.
```

Comment:
Might have to change things a bit here. I will know by the end of the day/when PR https://github.com/nrfconnect/sdk-nrf/pull/11209 is completed.